serverless/knative_eventing/serverless-kn-trigger.adoc

```diff
@@ -18,7 +18,7 @@ Before you can use triggers, you will need:
 * Knative Eventing and `kn` installed.
 * An available broker, either the `default` broker or one that you have created.
 +
-You can create the `default` broker either by following the instructions on xref:../serverless-using-brokers.adoc#serverless-using-brokers[Using brokers with Knative Eventing], or by using the `--inject-broker` flag while creating a trigger. Use of this flag is described in the procedure below.
+You can create the `default` broker either by following the instructions on xref:../../serverless/knative_eventing/serverless-using-brokers.adoc#serverless-using-brokers[Using brokers with Knative Eventing], or by using the `--inject-broker` flag while creating a trigger. Use of this flag is described in the procedure below.
```

Comment:
s/procedure below/following procedure/
**below** _adj_
Do not use to indicate a relative location in a document, as in “the information below.”